### ISSUE TITLE
Control vis without function e.g. changeView

### ISSUE THREAD

**Issue by m0uh85** [ioBroker/ioBroker.vis#705]
**Describe the bug**  
vis-2-beta.0.control.command "changeView" e.g. is not working

**To Reproduce**  
Try to send a command on vis-2-beta.0.control.command "changeView"

**Expected behavior**  
Change the View

**Versions:**  
v2.0.24

**Additional context**  
On VIS 1 everything is working fine.

**Comment by GermanBluefox** [comment]
It worked, but you should set the data and instance before. Now the values of data and instance will be taken at start

**Comment by m0uh85** [comment]
> It worked, but you should set the data and instance before. Now the values of data and instance will be taken at start

Sorry, but for me is it not working:
Set data to main/Licht e.g.
instance FFFFFF (should be all devices)
and then I send manually changeView - nothing happened

Edit:
It doens't matter what I'm doing at VIS2 
command
data
instance 
is not changing!

**Comment by GermanBluefox** [comment]
> FFFFFF

Too few 'F'. it must be `FFFFFFFF` or just `*`

**Comment by m0uh85** [comment]
> > FFFFFF
> 
> Too few 'F'. it must be `FFFFFFFF` or just `*`

I did a misstake when I wrote the upper Text bur in my script it is the right amount of F - Im using two Times the same script. ViS 1 is working. VIS 2 still not working.

When I open the Browser and change my view then at vis.0.control command/data/instance are changing 
When I do the same with my VIS 2 nothing happened at vis-2-beta.control command/data/instance ...
